python/Ganga/Core/GangaRepository/SessionLock.py

```diff
@@ -533,7 +533,7 @@ def session_read(self, fn):
             # This can fail (thats OK, file deleted in the meantime)
             fd = None
             try:
-                fd = os.open(fn, os.O_RDONLY)
+                fd = os.open(fn, os.O_RDWR)
```

Comment:
@egede I'm assuming this is needed here because we are touching the file to lock it on NFS?
Should `session_write` be modified to have read access as well? A user may not have seen this as a startup error as we don't write to lock files unless we're creating/saving a job.